### PR TITLE
Move off shakedown resolve conflicts

### PR DESCRIPTION
**Pull request by elezar**
@nickbp this fixes the conflicts.

### CODE REVIEW

File: testing/sdk_networks.py [hunk 1]
```diff
@@ -6,27 +6,43 @@
...
 
+
 log = logging.getLogger(__name__)
 
+
 ENABLE_VIRTUAL_NETWORKS_OPTIONS = {"service": {"virtual_network_enabled": True}}
 
 
+@retrying.retry(wait_fixed=1000, stop_max_delay=5 * 1000, retry_on_result=lambda res: not res)
+def _wait_for_endpoint_info(
```
Comment (elezar) [inline testing/sdk_networks.py]: Note that this adds retries (5 max) to all endpoint operations.

File: testing/sdk_networks.py [hunk 2]
```diff
@@ -41,20 +57,23 @@ def get_endpoint(package_name, service_name, endpoint_to_get, json=True) -> str:
...
+    """
+    # Catch if an empty string is passed in. Technically the command would succeed and return a list of endpoint names,
+    # but they should use get_endpoint_names() for this.
+    assert endpoint_name, "Missing endpoint_name. To get list of endpoint names, use get_endpoint_names()."
 
-    Meanwhile, custom service-defined endpoints may require json=False as they can contain free-form text."""
+    return _wait_for_endpoint_info(package_name, service_name, endpoint_name, True)
 
+
+def get_endpoint_string(package_name: str, service_name: str, endpoint_name: str) -> str:
```
Comment (elezar) [inline testing/sdk_networks.py]: Note the *explicit* `_string` function that also strips off whitespace.

File: frameworks/kafka/tests/client.py [hunk 3]
```diff
@@ -28,7 +28,7 @@ def __init__(self, service_options: dict):
         self._service_name = service_options["service"]["name"]
 
     def get_zookeeper_endpoint(self) -> str:
-        zookeeper_str = sdk_networks.get_endpoint(self._package_name, self._service_name, "zookeeper", json=False)
+        zookeeper_str = sdk_networks.get_endpoint_string(self._package_name, self._service_name, "zookeeper")
         return zookeeper_str.strip()
```
Comment (nickbp) [inline frameworks/kafka/tests/client.py]: Could remove the strip call here since get_endpoint_string is doing it automatically

**Comment by nickbp** [comment]
Will fix any issues in `move-off-shakedown`. It'll need a rebase anyway.

**Comment by elezar** [comment]
Thanks ...

**Pull request merged**